src/components/RichTextEditor/utils/getTextWidth.ts

```diff
@@ -0,0 +1,23 @@
+export function getTextWidth(text: string, font: string): number | null {
+    const canvas = document.createElement("canvas");
+    const context = canvas.getContext("2d");
+    if (context) {
+        context.font = font;
+        const metrics = context.measureText(text);
+
+        return Math.ceil(metrics.width * 1.05); // add a 5% safety margin
```

Comment:
Could you put an explanation of why we need it (need to load font first, this is based on first fallback, ...)

Reply:
Added